### PR TITLE
Add support for Stale bot

### PR DESCRIPTION
**Pull request by Arkni**
I will install the bots on this repo after we agree on the message of the comments.

Ref #2127

/cc @staabm

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - bug
+  - "MERGE ME"
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as idle and stale because it hasn't
+  had any recent activity. It will be automtically closed if no further activity
+  occurs. If you think this is wrong, or the problem still persists, just pop
+  a reply in the comments and @staabm or @Arkni will (try!) to follow up.
```
Comment (staabm) [inline .github/stale.yml]: please dont mention us... the bot will spam us with notifications ;-)

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - bug
+  - "MERGE ME"
```
Comment (staabm) [inline .github/stale.yml]: should we add "help wanted" here?

Comment (staabm) [reply .github/stale.yml]: and also "NEEDS REVIEW"?

File: .github/stale.yml (hunk 1 above)
Comment (Arkni) [reply .github/stale.yml]: Haha, I forgot why I put them in there.

I will replace them with `one of the maintainers`

File: .github/stale.yml (hunk 2 above)
Comment (Arkni) [reply .github/stale.yml]: You're right. I should add those too.

**Comment by Arkni** [comment]
@staabm Updated per above suggestions

**Comment by Arkni** [comment]
As far as I know, none of the two bots support a rollback action.

The `no-response` bot only support 3 options, which we already use. As for the `Stale` bot, there are still some usefull options to use. For example, `exemptMilestones` that can be used to ignore issues or pull requests that are part of a milestone. We can also use `only` option to limit the to issues only or pull requests (by default, it manage the two).

For more options, see https://github.com/probot/stale#usage

**Comment by Arkni** [comment]
Also, forgot to mention that the `no-response` bot has a annoying issue when the original poster closed the issue, see https://github.com/probot/no-response/issues/16

**Comment by staabm** [comment]
maybe we should start with the "stale" bot only and see how things goes.

the other bot config file could be stored in a separate PR for later merge.

**Comment by Arkni** [comment]
Sounds good to me. I will update the PR later today.

**Comment by Arkni** [comment]
Update per above suggestion. I think we are good to go.

**Comment by Arkni** [comment]
I broke the PR into two:
- This for `Stale` bot
- #2178 for `No-response` bot

**Comment by staabm** [comment]
ok cool, thank you.

**Comment by Arkni** [comment]
I have sent an installation request to `jquery-validation`. As you're the one who created this org, only you have the right to accept the request.

**Comment by jzaefferer** [comment]
Should the bot apply to all repos, or only this one?

**Comment by staabm** [comment]
we don't have "Issues"-tab activated in the other reposes

**Comment by jzaefferer** [comment]
I guess you already approved it? I'm not sure based on the email notification and Apps UI...

**Comment by staabm** [comment]
yep, I did.

**Pull request merged**